tools/src/test/java/org/apache/kafka/tools/AbstractResetIntegrationTest.java

```diff
@@ -246,7 +244,7 @@ public void testReprocessingFromScratchAfterResetWithoutIntermediateUserTopic(fi
         final List<KeyValue<Long, Long>> resultRerun = IntegrationTestUtils.waitUntilMinKeyValueRecordsReceived(resultConsumerConfig, OUTPUT_TOPIC, 10);
         streams.close();
 
-        assertThat(resultRerun, equalTo(result));
+        assertEquals(resultRerun, result);
```

Comment:
This `assertEquals` statement has the expected and actual values in the wrong order.